pilot/pkg/networking/core/v1alpha3/httproute.go

```diff
@@ -524,3 +476,56 @@ func getUniqueAndSharedDNSDomain(fqdnHostname, proxyDomain string) (string, stri
 	sharedSuffixes := strings.Join(reverseArray(sharedSuffixesInReverse), ".")
 	return uniqHostame, sharedSuffixes
 }
+
+func buildPassthroughVirtualHost(node *model.Proxy) *route.VirtualHost {
```

Comment:
This name is confusing as it can be either passthrough or blackhole

Reply:
Also I think this is unrelated

Reply:
How about `buildCatchAllVirtualHost` or `buildAllowAnyOrBlockAllVirtualHost` ?